README.md

```diff
@@ -105,18 +105,20 @@ Usage:
     <imgref>
 
 Flags:
+      --chown string    chown the ouput directory to match the specified UID:GID
```

Comment:
Isn't the real fix here for osbuild to stop trying to invoke chown() on its own in general?

If we need to chown, I'd advocate for automatically matching the uid/gid of the target output directory by default (I don't think osbuild does that).

Reply:
The issue here is that we want to run this command on top of a rootless user via sudo or via the API server, and need a way to allow the rootless user to use the disk images after they are created.

```
podman build ...
podman push ...
sudo podman run BIB -v output:/output --chown $UID:$GID 
podman manifest add output/IMAGE.NAME
```

Currently I have hacked up scripts to do `sudo chown`